### ISSUE TITLE
Thumbs in insight read-only

### ISSUE THREAD

**Issue by pwalczysko** [ome/omero-server#97]
As established during testing of https://github.com/ome/omero-server/pull/91#issuecomment-627429210, on  a read-only server, the thumbnails in insight are not displayed - instead, the placeholders are shown only.
The full viewer works as expected though.
The same server, user and image, but in read-write mode, displays the thumbnails correctly.

![Screen Shot 2020-05-12 at 16 18 32](https://user-images.githubusercontent.com/2478303/81729273-a2223980-9483-11ea-82b4-adb27f45113c.png)

cc @mtbc

**Comment by joshmoore** [comment]
And after displaying the images in read-write, do the thumbnails then appear from the read-only server?

**Comment by pwalczysko** [comment]
> And after displaying the images in read-write, do the thumbnails then appear from the read-only server?

Sorry, no idea what do you mean.
The server is either configured as read-write, or read-only. 
In read-write mode, there are thumbs.
In read-only mode, there are no thumbs.
Your sentence supposes some causality and mixing of states, so don't understand ?

**Comment by joshmoore** [comment]
> Possibly you mean that the thumbs were somehow never there after import

That was my hypothesis, yes. Thanks.

(Note: your subsequent edit text seems to mix RO & RW)

**Comment by pwalczysko** [comment]
> (Note: your subsequent edit text seems to mix RO & RW)

Indeed, thank you , corrected and added one more sentence to https://github.com/ome/omero-server/issues/97#issuecomment-627522800
